### PR TITLE
Release 0.5.7

### PR DESCRIPTION
**Pull request by blootsvoets**
Changes since version 0.5.6:
- Fix liquibase changelog
- Added retry action for records
- Assert that all applicable CSV processors are used, otherwise the processing fails
- Added client-side caching
- Updated frontend dependencies

### CODE REVIEW

File: radar-upload-backend/src/main/java/org/radarbase/upload/resource/ProjectResource.kt [hunk 1]
```diff
@@ -44,11 +45,13 @@ class ProjectResource(
 
     @GET
     @NeedsPermission(Permission.Entity.PROJECT, Permission.Operation.READ)
+    @Cache(maxAge = 300, isPrivate = true)
```
Comment (nivemaham) [inline radar-upload-backend/src/main/java/org/radarbase/upload/resource/ProjectResource.kt:48]: This is a new feature from radar-jersey. Nice. Could you please tell me what does the `isPrivate `property in the annotation do?

File: build.gradle.kts [hunk 2]
```diff
@@ -7,19 +7,19 @@ plugins {
 
 subprojects {
     group = "org.radarbase"
-    version = "0.5.6"
+    version = "0.5.7-SNAPSHOT"
```
Comment (nivemaham) [inline build.gradle.kts:10]: Shouldn't we make it 0.5.7?

**Review by nivemaham — APPROVED** [review]
The backend version should be moved out of the SNAPSHOT. Otherwise, LGTM.

File: radar-upload-backend/src/main/java/org/radarbase/upload/resource/ProjectResource.kt (hunk 1 above)
Comment (blootsvoets) [reply radar-upload-backend/src/main/java/org/radarbase/upload/resource/ProjectResource.kt:48]: It is a client header, that tells the client (e.g., the browser) that it should not cache the same data for other users, but only for the current session: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#cacheability

**Pull request merged**